.travis.yml

```diff
@@ -14,8 +12,6 @@ matrix:
         python setup.py test
         # https://github.com/tholo/pytest-flake8/issues/27
         flake8 bin/lsst-build
```

Comment:
Why isn't this checking the other python code?

Reply:
see the comment above -- I couldn't figure out how to get it to check one file without a `.py` extension.

Reply:
Which is fine, but I think from what I see there that we don't then test any of the other python code. Adding a second call to flake8 is needed I think (or adding `python/` to the arguments for that flake8 call).

Reply:
All of the python code is flake8'd by `pytest-flake8`, only the bin is being missed and that should go away if it's converted to a console script.

Reply:
How do you explain the flake8 problems on master then?

Reply:
flake8 wasn't complaining *a year ago* when master was last built.  flake8 output is visible in the console of the most recent build: https://travis-ci.org/lsst/lsst_build/jobs/339213143